scripts/cluster/wrapper.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Maybe it is cumbersome to have an own wrapper.sh if the only thing it does is pip install . (which could be easily included into the other scripts? Or maybe call the wrapper installer?

Reply:
not sure either. usually we had 2 scripts - one for the srun command with predefined standard args, and one for the "set up" of the VM / conda env. so the usual call to a script was like "srun-script [additional-cl-args-for-slurm] setup-script python my-evaluation-script parameters". keeping batch.sh and wrapper/installer.sh separate allows to add args to batch.sh, before wrapper.sh is run. but I guess we could move "pip install ." to the task-specific scripts, but that would duplicate it :)